protocol/transport/envelopes_monitor.go

```diff
@@ -1,11 +1,27 @@
 package transport
 
 import (
+	"context"
+	"errors"
+	"sync"
+
 	"go.uber.org/zap"
 
 	"github.com/status-im/status-go/eth-node/types"
 )
 
+// EnvelopeState in local tracker
+type EnvelopeState int
+
+const (
+	// NotRegistered returned if asked hash wasn't registered in the tracker.
+	NotRegistered EnvelopeState = -1
+	// EnvelopePosted is set when envelope was added to a local waku queue.
+	EnvelopePosted EnvelopeState = iota
```

Comment:
It is best to avoid zero value consts as they can collide with nil value ints.

```suggestion
	EnvelopePosted EnvelopeState = iota + 1
```